### PR TITLE
docs: clarify imports for useTable core examples

### PR DESCRIPTION
**Pull request by jamesthesken**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

_Any change needs to be discussed before proceeding. Failure to do so may result in the rejection of the pull request._

Please provide enough information so that others can review your pull request:
This PR adds import statements to the `useTable` documentation under `Core API/Hooks/useTable`, as some properties are only available with `import { useTable } from "@refinedev/core"`. For example, changing examples such as:

```tsx
useTable({
    pagination: {
        pageSize: 20,
    },
});
``` 

to:

```tsx
import { useTable } from "@refinedev/core"
useTable({
    pagination: {
        pageSize: 20,
    },
});
``` 

Explain the **details** for making this change. What existing problem does the pull request solve?

It was confusing for myself while using the useTable hook referencing the examples without clarifying which packages to import.

### Test plan (required)

Demonstrate the code is solid. If not, please add `WIP:` in its title.

### Closing issues

Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### Self Check before Merge

Please check all items below before review.

-   [x] Corresponding issues are created/updated or not needed
-   [x] Docs are updated/provided or not needed
-   [x] Examples are updated/provided or not needed
-   [x] TypeScript definitions are updated/provided or not needed
-   [x] Tests are updated/provided or not needed
-   [x] Changesets are provided or not needed

### CODE REVIEW

**Comment by jamesthesken** [comment]
Closes: [#5216](https://github.com/refinedev/refine/issues/5216)

**Review by omeraplak — APPROVED** [review]
Thank you! 💥

File: documentation/docs/api-reference/core/hooks/useTable/index.md [hunk 1]
```diff
@@ -94,6 +94,7 @@ method={{
 />
 
 ```tsx
+import { useTable } from "@refinedev/core"
```
Comment (omeraplak) [inline documentation/docs/api-reference/core/hooks/useTable/index.md:97]:
```suggestion
import { useTable } from "@refinedev/core";
```

Could you add missing semicolons?

**Review by omeraplak — CHANGES_REQUESTED** [review]

**Review by alicanerdurmaz — APPROVED** [review]

File: documentation/docs/api-reference/core/hooks/useTable/index.md (hunk 1 above)
Comment (jamesthesken) [reply documentation/docs/api-reference/core/hooks/useTable/index.md]: Thanks for reviewing I just added and pushed a new commit😄

**Review by omeraplak — APPROVED** [review]

**Pull request merged**